src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -311,12 +313,18 @@ export default [
         icon: Expensicons.LinkCopy,
         successIcon: Expensicons.Checkmark,
         successTextTranslateKey: 'reportActionContextMenu.copied',
-        shouldShow: (type, reportAction, isArchivedRoom, betas, menuTarget) => {
+        shouldShow: (type, reportAction, isArchivedRoom, betas, menuTarget, isChronosReport, reportID, isPinnedChat, isUnreadChat, isOffline, isMini) => {
             const isAttachment = ReportActionsUtils.isReportActionAttachment(reportAction);
 
             // Only hide the copylink menu item when context menu is opened over img element.
             const isAttachmentTarget = lodashGet(menuTarget, 'tagName') === 'IMG' && isAttachment;
-            return Permissions.canUseCommentLinking(betas) && type === CONST.CONTEXT_MENU_TYPES.REPORT_ACTION && !isAttachmentTarget && !ReportActionsUtils.isMessageDeleted(reportAction);
+            return (
+                !isMini &&
```

Comment:
Could you please help me understand why all icons now depend on `isMini`?

Reply:
@rushatgabhane For the changes we need to selectively show some option only for `mini` menu. For example, the `ThreeDotMenu` should be only visible when mini context menu is active and not for the full.